Pipfile

```diff
@@ -27,10 +27,10 @@ maya = "*"
 #
 # Third-Party Ethereum
 #
-eth-utils="==1.2.2"
+eth-utils="==1.2.1"
```

Comment:
Why is this? AFAIK, v1.2.2 was working.

Reply:
It causes an unresolvable sub-dependency condition with `eth-utils`